build/Nuget/Microsoft.VisualStudio.ProjectSystem.VSInsertionNuget.proj

```diff
@@ -1,16 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\Targets\Settings.targets" />
-  
+  <ImportGroup>
+    <Import Project="..\Targets\Settings.targets" />
+    <Import Project="..\Targets\VSL.Versions.targets" />
+  </ImportGroup>
+
   <PropertyGroup>
-    <NuGetVersion>1.0.0.0</NuGetVersion>
-    <NuGetVersionType>InsertionPurposeOnly</NuGetVersionType>
-    <NuGetOutDir>$(OutDir)NuGet\$(NuGetVersionType)\</NuGetOutDir>
+    <NuGetOutDir>$(OutDir)DevDivPackages\Roslyn\</NuGetOutDir>
```

Comment:
The Roslyn insertion tool has this path hard-coded in it hence this name. There is not much we can do here